src/transport/SessionManager.cpp

```diff
@@ -401,6 +401,26 @@ Optional<SessionHandle> SessionManager::AllocateSession(uint16_t sessionId)
     return mSecureSessions.CreateNewSecureSession(sessionId);
 }
 
+CHIP_ERROR SessionManager::InjectPaseSessionWithTestKey(SessionHolder & sessionHolder, uint16_t localSessionId, NodeId peerNodeId,
```

Comment:
As a followup: should this inject a PASE session or CASE session?  The old thing did PASE, but it also predated CASE even existing as I recall.